integration/js/README.md

```diff
@@ -142,12 +142,33 @@ Run the test:
 ```shell
 $KITE_HOME/scripts/mac/path/r configs/<test_name>.config.json
 ```
-Steps to add test file to Sauce Labs virtual machine
-To create a fake video or audio stream you need the media file on Sauce Labs virtual machine before running your test. For that you can use a pre-run executable script to download file from a public storage to the virtual machine running your tests. 
-- Create a pre-run script by following the steps given on this link: https://docs.saucelabs.com/web-apps/automated-testing/selenium/pre-run-executables/#downloading-files-to-a-vm-prior-to-testing. 
-    On sauce labs virtual machines files are stored in the below folder locations
-    Windows = C:\Users\Administrator\Downloads
-    Mac = /Users/chef/Downloads
-    Linux = /home/chef/Downloads
-- Upload the pre-run executable to Sauce Labs storage by using Sauce Labs API methods: https://docs.saucelabs.com/dev/api/storage/
-- Add the pre-run to Sauce configs in your code like this prerun: 'storage:filename=filename.ext'. You can also use storage id of the file location. 
+## Testing against custom audio/video stream in Sauce Labs
+Some tests might require testing against a custom audio/video stream. To test against a custom audio or video stream, you need the media file on Sauce Labs virtual machine before running your test. For that you can use a pre-run executable script to download media file from a public storage to the virtual machine before running your tests. 
```

Comment:
- nit: `a custom audio or video stream` => `a custom audio/video stream` for consistency with the title and the first sentence.
- `For that you can` => Add a comma `For that, you can`. Or you can use `To do so, you can` (More common in AWS docs)
- `a public storage` => `public storage`
- nit: This paragraph repeats `against a custom audio/video stream` and `before running your test`.  You can simplify this a bit but it's not a major blocker.

Reply:
Done